### PR TITLE
New demo: Integrate Score and CI without a connector

### PR DESCRIPTION
**Pull request by ccoeurderoy**
### Description

- Create a little express server rendering an index.html file with a button to execute the score and display the result

### CODE REVIEW

File: score-integration-with-connector/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@algoan/score-integration-poc",
+  "name": "@algoan/score-integration-with-connector",
```
Comment (qhdinh) [inline score-integration-with-connector/package.json:2]: *without*

**Review by meriamBenSassi — APPROVED** [review]
This is very useful !

File: score-integration-without-connector/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Install all dependencies running:
+
+```bash
+npm install
+```
+
+## Usage
+
+Once your local connector is launched and your REST hooks are created, you can now start the application:
```
Comment (qhdinh) [inline score-integration-without-connector/README.md]: This app uses hard-coded data, why does it require a local connector et REST hook?

**Review by qhdinh — APPROVED** [review]

File: score-integration-with-connector/package.json (hunk 1 above)
Comment (ccoeurderoy) [reply score-integration-with-connector/package.json:2]: Not for this one 😉

File: score-integration-without-connector/README.md (hunk 2 above)
Comment (ccoeurderoy) [reply score-integration-without-connector/README.md]: Wrong copy-paste, thank you!

File: score-integration-with-connector/package.json (hunk 1 above)
Comment (qhdinh) [reply score-integration-with-connector/package.json:2]: Ah I mislooked 😄  it's a different directory.

**Pull request merged**